Assets/Scripts/InGameUI/PlaceUnitsController.cs

```diff
@@ -68,19 +77,23 @@ public class PlaceUnitsController : ParentController {
 
 	// Action when submit button is pressed
 	void PlaceUnits(){
-		Dictionary<string, object> response = Server.PlaceUnits(GameData.CurrentMatch);
-		if(Parse.Bool(response["Success"])){
-			GameController.PlacingUnits = false;
-			for(int x = 0; x < GameController.Tokens.Length; x++) {
-				for(int y = 0; y < GameController.Tokens[x].Length; y++) {
-					GameController.Tokens[x][y].SetActionProperties("clear");
-				}
-			}
-			Destroy(gameObject);
-		}
-		else{
+		Server.PlaceUnits(GameData.CurrentMatch, this);
```

Comment:
Should we add the loading circle here too?  Just in case the server call takes longer than normal?